src/components/Comment/Comment.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { CommentType } from '../types/CommentType';
+import './Comment.scss';
+
+export const Comment = ({ body, email, name }) => (
+  <>
+    <hr />
+    <p>{body}</p>
+    <div className="comment__user">
+      <span>{name}</span>
+      <span>{email}</span>
+    </div>
+  </>
+);
+
+Comment.propTypes = CommentType;
```

Comment:
pls do the same for Post and User

```suggestion
Comment.propTypes = {
  body: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired,
  email: PropTypes.string.isRequired,
};
```